### PR TITLE
Added function that returns the name of a currency.

### PR DESCRIPTION
**Pull request by AntonCroner**

### CODE REVIEW

**Review by marcushanses — DISMISSED** [review]

File: Currency.ts [hunk 1]
```diff
@@ -596,4 +596,188 @@ export namespace Currency {
...
+			SBD: "Solomon Islands dollar",
+			SCR: "Seychelles rupee",
+			SDG: "Sudanese pound",
+			SEK: "Swedish krona",
+			SGD: "Singapore dollar",
+			SHP: "Saint Helena pound",
+			SOS: "Somalian shilling",
+			SRD: "Surinamese dollar",
+			SSP: "South Sudanese pound",
+			STN: "15] 	São Tomé and Príncipe dobra",
```
Comment (Magnus93) [inline Currency.ts]: `15]    `?

File: Currency.ts [hunk 2]
```diff
@@ -596,4 +596,188 @@ export namespace Currency {
...
+			ISK: "Icelandic króna",
+			JMD: "Jamaican dollar",
+			JOD: "Jordanian dinar",
+			JPY: "Japanese yen",
+			KES: "Kenyan shilling",
+			KGS: "Kyrgyzstani som",
+			KHR: "Cambodian riel",
+			KMF: "Comoro franc",
+			KPW: "North Korean won",
+			KRW: "e] 	South Korean won",
```
Comment (Magnus93) [inline Currency.ts]: `e]   `?

File: Currency.ts [hunk 3]
```diff
@@ -596,4 +596,188 @@ export namespace Currency {
...
+			KZT: "Kazakhstani tenge",
+			LAK: "Lao kip",
+			LBP: "Lebanese pound",
+			LKR: "Sri Lankan rupee",
+			LRD: "Liberian dollar",
+			LSL: "Lesotho loti",
+			LYD: "Libyan dinar",
+			MAD: "Moroccan dirham",
+			MDL: "Moldovan leu",
+			MGA: "f] 	Malagasy ariary",
```
Comment (Magnus93) [inline Currency.ts]: `f]   `?

File: Currency.ts [hunk 4]
```diff
@@ -596,4 +596,188 @@ export namespace Currency {
...
+			MKD: "Macedonian denar",
+			MMK: "Myanmar kyat",
+			MNT: "Mongolian tögrög",
+			MOP: "Macanese pataca",
+			MRU: "f][10] 	Mauritanian ouguiya",
```
Comment (Magnus93) [inline Currency.ts]: `f][10]    `?

Comment (AntonCroner) [reply Currency.ts]: \+

File: Currency.ts (hunk 3 above)
Comment (AntonCroner) [reply Currency.ts]: \+

File: Currency.ts (hunk 2 above)
Comment (AntonCroner) [reply Currency.ts]: \+

File: Currency.ts (hunk 1 above)
Comment (AntonCroner) [reply Currency.ts]: \+

**Review by Magnus93 — APPROVED** [review]

**Pull request merged**